src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1299,6 +1299,18 @@ public static double Round(double value, int digits)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static double Round(double value, MidpointRounding mode)
         {
+            // Inline the most common cases
+            if (mode == MidpointRounding.ToEven)
+            {
+                return Round(value);
+            }
```

Comment:
If this is correct, why is the `Round(double, int)` method above implemented as `return Round(value, digits, MidpointRounding.ToEven);` rather than `return Round(value);`... and why does it even exist?

Reply:
And other than that delegation, is it common to pass ToEven here?  Seems like this case should be deleted.

Reply:
Ugh, sorry, I completely misread the signature... I thought digits was being passed in.  Ignore the first comment.  Second question still stands.

Reply:
I think it's not common to pass anything other than ToEven or AwayFromZero but let me use IsConstArgument since I am not ready to prove my words 🙂

Reply:
> I think it's not common to pass anything other than ToEven or AwayFromZero

AwayFromZero makes sense: for most of MidpointRounding's existence, the only two values were ToEven and AwayFromZero.  But ToEven is the default, so it seems like it'd be relatively rare to call Round(value, ToEven) when you can just do Round(value).

Reply:
> but let me use IsConstArgument

Is that different from IsKnownConstant?

Reply:
> > but let me use IsConstArgument
> 
> Is that different from IsKnownConstant?

I always forget the name, today I tried to write it as `IsJitConstant` 😄